docs/en/integrations/language-clients/nodejs.md

```diff
@@ -798,6 +799,47 @@ for [basic](https://github.com/ClickHouse/clickhouse-js/blob/main/examples/basic
 and [mutual](https://github.com/ClickHouse/clickhouse-js/blob/main/examples/mutual_tls.ts)
 TLS in the repository.
 
+## Keep Alive
+
+By default, client enables Keep-Alive in the underlying HTTP agent. 
+If you are experiencing `socket hang up` errors, there are several options to resolve this issue:
+
+* Increase the value of the [keep_alive_timeout](https://clickhouse.com/docs/en/operations/server-configuration-parameters/settings#keep-alive-timeout) ClickHouse server setting in `config.xml`, as by default it could be as little as 3s.
```

Comment:
```suggestion
* Increase the value of the [keep_alive_timeout](/docs/en/operations/server-configuration-parameters/settings.md/#keep-alive-timeout) ClickHouse server setting in `config.xml`, as by default it could be as little as 3s.
```